### ISSUE TITLE
Changing seeds

### ISSUE THREAD

**Issue by jabr** [marcelklehr/smokesignal#19]
In a long running application with turnover and new deploys, the seed nodes could change.

I think a mechanism for updating the seeds in active modes would be useful.

**Comment by jabr** [comment]
And to clarify, I'm happy to add that feature. Just looking for feedback on how everyone would like it work.

**Comment by marcelklehr** [comment]
Agreed. I think this should be rather simple to implement as all parts that deal with seeds take them directly from the options, afaict.

**Comment by downitch** [comment]
Hello, guys! Is the feature implemented already?

**Comment by marcelklehr** [comment]
Hey :) Noone has worked on it, so far. I'm rather busy with other projects, but I'm happy to accept pull requests :)
